src/components/Slices/SlicesTable.jsx

```diff
@@ -1,56 +1,114 @@
 import React from "react";
 import PropTypes from "prop-types";
 import MaterialTable from "material-table";
-import { Typography, Box } from "@material-ui/core";
+import { Typography, Box, makeStyles } from "@material-ui/core";
 import { satoshisToBitcoins } from "unchained-bitcoin";
 
+import {
+  compareSlicesByPath,
+  compareSlicesByBalance,
+  compareSlicesByUTXOCount,
+  compareSlicesByTime,
+} from "../../utils/slices";
+
+// Components
 import Copyable from "../Copyable";
 import { slicePropTypes, clientPropTypes } from "../../proptypes";
 import SliceDetails from "./SliceDetails";
 
-const AddressTable = ({ slices, search, paging, title, client, network }) => {
+const useStyles = makeStyles((theme) => ({
+  panel: {
+    boxShadow: `inset 0px 0px 4px 1px ${theme.palette.grey[500]}`,
+  },
+  spent: {
+    textDecoration: "line-through",
+  },
+}));
+
+const SlicesTable = ({
+  slices,
+  search,
+  paging,
+  title,
+  client,
+  network,
+  disabled,
+}) => {
+  const classes = useStyles();
+
   const options = {
     search,
     paging,
     detailPanelType: "single",
     showTitle: (title && title.length) || false,
+    emptyRowsWhenPaging: false,
+    pageSize: 10,
+    pageSizeOptions: [10, 20, 50],
   };
   options.toolbar = options.showTitle && options.search;
+  let columns = [
```

Comment:
Added a new column for the address list view. Further down is a filter to remove columns that are "disabled"

Reply:
It's not a UI option, just an option in the component itself. So `SlicesTable` is used in two places in the app now, receive and address list. for the latter, there is no "Time" column.